playbooks/provisioning/openstack/sample-inventory/group_vars/all.yml

```diff
@@ -0,0 +1,42 @@
+env_id: "openshift"
+openstack_dns_domain: "example.com"
+openstack_nameservers: ["192.168.1.1"]
```

Comment:
the fix for networkmanager config that worked for me was using:
```
- "{{ hostvars[groups.dns|first].openstack.private_v4 }}"
- 8.8.8.8
```
This allows the nodes to resolve both external and internal names, with a priority for the intranet-first